docs/sdk/examples/pastefromlibreoffice.html

```diff
@@ -130,7 +130,7 @@ <h1>Pasting content from LibreOffice Writer <a class="documentation" href="https
 						&lt;/tbody&gt;
 					&lt;/table&gt;
 
-					&lt;p&gt;&lt;span style="font-family:Georgia,serif"&gt;The project that you participated in is of utmost importance to the future success of our platform. We are very proud to share that the CKEditor implementation was a huge success and brought congratulations from both the key Stakeholders and the Customers:&lt;/span&gt;&lt;/p&gt;
+					&lt;p&gt;&lt;span style="font-family:Georgia,serif"&gt;The project that you participated in is of utmost importance to the future success of our platform. We are very proud to share that the CKEditor 4 implementation was a huge success and brought congratulations from both the key Stakeholders and the Customers:&lt;/span&gt;&lt;/p&gt;
```

Comment:
```suggestion
					&lt;p&gt;&lt;span style="font-family:Georgia,serif"&gt;The project that you participated in is of utmost importance to the future success of our platform. We are very proud to share that the CKEditor implementation was a huge success and brought congratulations from both the key Stakeholders and the Customers:&lt;/span&gt;&lt;/p&gt;
```

This content mimics the attached sample file so let's leave it as is.